asserts/pool.go

```diff
@@ -253,6 +335,19 @@ func (p *Pool) curRevision(ref *Ref) (int, error) {
 	return RevisionNotKnown, nil
 }
 
+func (p *Pool) curSeqRevision(seq *AtSequence) (int, error) {
+	if seq.Pinned {
```

Comment:
I'm not sure this pinned is right,  as long a seq has Sequence we can find what is current, even if ultimately we are looking for the latest